src/analytics/Properties.tsx

```diff
@@ -265,6 +265,8 @@ interface KeylessBackupEventsProperties {
   [KeylessBackupEvents.cab_progress_completed_continue]: undefined
   [KeylessBackupEvents.cab_progress_failed_later]: undefined
   [KeylessBackupEvents.cab_progress_failed_manual]: undefined
+  [KeylessBackupEvents.cab_progress_failed_manual_onboarding]: undefined
```

Comment:
Can we pass an origin instead of having `cab_progress_failed_manual` and `cab_progress_failed_manual_onboarding`.

```
  [KeylessBackupEvents.cab_progress_failed_manual]: {
    origin: 'onboarding' | 'settings'
  }
```